calibrate.py

```diff
@@ -822,6 +840,7 @@ def calibrate(self):
         print("Starting image processing")
         stereo_calib = calibUtils.StereoCalibration()
         stereo_calib.traceLevel = self.args.traceLevel
```

Comment:
Defile this properly inside it and set it through init or calibrate

Reply:
Resolved by adding a new branch and defining it in init.